python/pynessie/auth/aws.py

```diff
@@ -15,9 +15,9 @@
 # limitations under the License.
 #
 """Use AWS4Auth and botocore to fetch credentials and sign requests."""
-from botocore.credentials import get_credentials
-from botocore.session import Session
-from requests_aws4auth import AWS4Auth
+from botocore.credentials import get_credentials  # type: ignore
```

Comment:
Is this ignore for mypy?
```

Reply:
yes this is a more specific `--ignore-missing-imports`

Reply:
Can't we just add this in the `setup.cfg` instead, e.g:
```
[mypy-botocore.*]
ignore_missing_imports = True
```

Reply:
ok i have pushed another commit to also ignore botocore imports via the config file (i used 3 usage spots as the cutoff point before).